.drone.yml

```diff
@@ -86,6 +86,11 @@ steps:
   commands:
   - ./.ci/travis/travis.sh javac15
 
+- name: javac16
+  image: maven:3.6.3-adoptopenjdk-16
```

Comment:
Just curious, why this image and not `maven:3.8.1-openjdk-16` ?

Reply:
Good question. Changed to 3.8.1.
If this version turns out to be incompatible, I will revert it back and create a separate issue.